### DISCUSSION TITLE
I want to be able to use custom fonts on Vitepress. What should I do?

### DISCUSSION THREAD

**Discussion by bd-dxg** [vuejs/vitepress#1630] [Q&A] [answered]
I want to be able to use custom fonts on Vitepress. What should I do?

Font files may be in a local or CDN environment

**Comment by brc-dd** [comment, marked as answer]
https://www.freecodecamp.org/news/how-to-build-a-modern-documentation-site-with-vitepress/#how-to-use-custom-fonts
https://vitepress.vuejs.org/guide/theme-introduction#customizing-css
